pos/modules/checkout/hooks/useTDB.tsx

```diff
@@ -27,7 +27,7 @@ const useTDB = () => {
 }
 
 export const useTDBTransaction = (options: {
-  onCompleted: () => void
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider refining the type definition for `onCompleted` parameter from `any`.

```suggestion
onCompleted: (info?: { [key: string]: any }) => void
```